care/facility/migrations/0483_migrate_patient_consultations.py

```
@@ -0,0 +1,827 @@
+                encounter_class=suggestion_to_class_map.get(
+                    consultation.suggestion, "amb"
+                ),
+                created_by_id=consultation.created_by_id,
+                updated_by_id=consultation.last_edited_by_id,
+                created_date=consultation.created_date,
+                modified_date=consultation.modified_date,
+                current_location_id=current_location,
+            )
+
+            for cbed in consultation.consultationbed_set.all():
+                location = FacilityLocation._base_manager.get(
```

Ruff (0.8.2)

Private member accessed: `_base_manager`

(SLF001)